lib/context_adapter_helper.js

```diff
@@ -23,7 +23,9 @@
 
 'use strict';
 
+var caLogger = require('logops');
 var caConfig = require('./context_adapter_configuration');
+caLogger.setLevel(caConfig.LOGOPS_LEVEL);
```

Comment:
I suggested a place below, but, reading your answer, I tend to think that you might have misunderstood how `require()` works. Did you know that modules are singletons? That the `require()` function executed n times creates a single object and returns it? You can leave the all the `require()` calls where they are and make a single centralized call to set the log level.